docs/appendices/release-notes/5.7.0.rst

```diff
@@ -133,3 +133,8 @@ External contributions
   operator as syntax sugar for the :ref:`scalar-power` scalar function. E.g.::
 
     SELECT 5 ^ 2;
+
+- `Dhruv Patel <https://github.com/DHRUV6029>`_ added support for the 'strpos(string, substring)'
+  operator for the :ref:`scalar-string` scalar function. E.g.::
+
+    SELECT strpos('crate' , 'ate');
```

Comment:
```suggestion

- `Dhruv Patel <https://github.com/DHRUV6029>`_ added support for the
  :ref:`strpos <scalar-strpos>` scalar function.    
```

Reply:
Fixed reference and also dropepd the example, function name is pretty much self explaining.